markets/perps-market/contracts/modules/PerpsMarketFactoryModule.sol

```diff
@@ -64,6 +66,10 @@ contract PerpsMarketFactoryModule is IPerpsMarketFactoryModule {
     ) external override returns (uint128) {
         FeatureFlag.ensureAccessToFeature(_CREATE_MARKET_FEATURE_FLAG);
 
+        if (marketOwner == address(0)) {
```

Comment:
doesn't solidity automatically check that you're not sending address(0)?

Reply:
no, if I remove that if, it will allow the function to execute.